api/autonomous_car/routes.py

```diff
@@ -0,0 +1,24 @@
+from flask import jsonify
+from flask import render_template
+
+from api.autonomous_car import controllers
+from app import app
+
+
+@app.route('/autonomous-car/')
+def automated_car_instructions():
+    return render_template('autonomous-car.html')
+
+
+@app.route('/autonomous-car/routes/')
+def automated_car_route():
+    return jsonify({
+        'car_route': controllers.get_reliable_car_route()
+    })
```

Comment:
Let's have more endpoints to get the different car routes so they can play around a bit more. This one is fine to return the basic route, but a couple others might include: 
`/autonomous-car/routes/basic/`  
`/autonomous-car/routes/out-of-bounds/`
`/autonomous-car/routes/with-obstacles/`